app/test_main.py

```diff
@@ -1,3 +1,61 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, result",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="initial conditions"
```

Comment:
it isn't descriptive.It's often beneficial to use more descriptive messages to communicate the purpose or context of the test case for better readability and understanding

Reply:
Fix in all places